testing/integration_testing/build_testapps.py

```diff
@@ -444,6 +460,55 @@ def _run(args, timeout=1200):
   return subprocess.run(args=args, timeout=timeout, check=True)
 
 
+@attr.s(frozen=True, eq=False)
+class VCPKG(object):
```

Comment:
Would be easier to read if class names were camel case. 
Reading on the client side, I initially confused this with a constant.

Reply:
Yeah I wasn't sure how to deal with the acronym, but I've seen a sensible rule of thumb that says 1-2 letters gets upper case (e.g. IO), while 3+ just capitalize the first letter (e.g. Unesco).

It also occurred to me while writing this that pkg is not an acronym. 

Regardless, I'll just go with Vcpkg.